### PR TITLE
chore/UI changes

### PR DESCRIPTION
**Pull request by mfshao**
Minor UI updates

### Improvements
- UI updates

### CODE REVIEW

File: data/config/ndef-qa.json [hunk 1]
```diff
@@ -223,7 +223,7 @@
         "options": []
       },
       { 
-        "title": "Genus species",
+        "title": "Genus Species",
```
Comment (gkuffel) [inline data/config/ndef-qa.json]: So for this one the word species should intentionally be left lower case as this is a biological naming convention. For example , Homo sapiens.

File: data/config/ndef-qa.json [hunk 2]
```diff
@@ -235,7 +235,7 @@
       { "field": "gender", "name": "Gender" },
       { "field": "ethnicity", "name": "Ethnicity" },
       { "field": "strain", "name": "Strain" },
-      { "field": "species", "name": "Genus species" },
+      { "field": "species", "name": "Genus Species" },
```
Comment (gkuffel) [inline data/config/ndef-qa.json]: leave species lower case.

File: data/config/ndef.json [hunk 3]
```diff
@@ -197,7 +197,7 @@
       },
       "species": {
         "chartType": "bar",
-        "title": "Genus species"
+        "title": "Genus Species"
```
Comment (gkuffel) [inline data/config/ndef.json]: leave lower-case

File: data/config/ndef.json [hunk 4]
```diff
@@ -223,19 +223,19 @@
         "options": []
       },
       { 
-        "title": "Species",
+        "title": "Genus Species",
```
Comment (gkuffel) [inline data/config/ndef.json]: lower-case

File: data/config/ndef.json [hunk 5]
```diff
@@ -223,19 +223,19 @@
...
     "fieldMapping" : [
-      { "field": "dataset", "name": "Dataset" },
+      { "field": "dataset", "name": "Resource" },
       { "field": "studyAccession", "name": "Study" },
       { "field": "phenotype", "name": "Phenotype" },
       { "field": "gender", "name": "Gender" },
       { "field": "ethnicity", "name": "Ethnicity" },
       { "field": "strain", "name": "Strain" },
-      { "field": "species", "name": "Genus species" },
+      { "field": "species", "name": "Genus Species" },
```
Comment (gkuffel) [inline data/config/ndef.json]: lower case

Comment (mfshao) [reply data/config/ndef.json]: thanks! all fixed in https://github.com/uc-cdis/data-ecosystem-portal/pull/12/commits/81b54b67b622c3dcb4e89d0b448d1f2f8028ae33

**Pull request merged**